src/main/java/net/dunice/MaxRecurringSymbol.java

```diff
@@ -0,0 +1,37 @@
+package net.dunice;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class MaxRecurringSymbol {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Введите фразу или слово: ");
+        String myMessage = scanner.nextLine();
+
+        System.out.println("Самый повторяющийся символ - " + selectSymbol(myMessage));
+    }
+    private static Character selectSymbol(String userMessage){
+        Map<Character,Integer> messageMap = new HashMap();
```

Comment:
некорректно работает
при вводе  ffaass 
Самый повторяющийся символ - a
хотя это f 

попробуй так 
```suggestion
        Map<Character,Integer> messageMap = new LinkedHashMap();
```